model/pipeline/src/main/proto/endpoints.proto

```diff
@@ -28,20 +28,38 @@ option go_package = "pipeline_v1";
 option java_package = "org.apache.beam.model.pipeline.v1";
 option java_outer_classname = "Endpoints";
 
+// A description of how to connect to a Beam API endpoint.
 message ApiServiceDescriptor {
   // (Required) The URL to connect to.
-  string url = 2;
+  string url = 1;
```

Comment:
I assume this proto message isn't in common use yet, right? Because changing the field number seems risky otherwise.

Reply:
Externally we will be consistent but I'll need to ensure that within Google I renumber this field atomically which I'm working on.

Reply:
Gotcha, sounds good.